xors-api/src/api/mod.rs

```diff
@@ -138,7 +139,12 @@ pub fn service(
                         .push(Router::with_path("signin").post(jwt::signin))
                         .push(Router::with_path("captcha").get(jwt::captcha)),
                 )
-                .push(Router::with_path("user").get(user::get_user_info)),
+                .push(Router::with_path("user").get(user::get_user_info))
+                .push(
+                    Router::with_path("game")
+                        .push(Router::with_path("lastest").get(game::get_lastest_games))
+                        .push(Router::with_path("<uuid>").get(game::get_game_by_uuid)),
```

Comment:
The routes will be like

- `/game/<uuid>`
- `/game/lastest`

This makes it confusing

Move `/game/lastest` to `/games` this will make it better